collectors/scm/github/src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java

```diff
@@ -32,6 +32,7 @@
 import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestOperations;
 
+
```

Comment:
Can you remove all of the formatting? There are unnecessary changes and can make things harder to read.